pr-data.csv

```diff
@@ -1666,13 +1666,13 @@ https://github.com/apache/incubator-shenyu,3777209fc76800175df3962213ca1febe57ed
 https://github.com/apache/incubator-streampipes,7ba9213012b72ea70edc6848c18ba6d90ff2c534,streampipes-connect,org.apache.streampipes.connect.adapter.format.json.xml.XmlTest.parseEventCarPark,ID,Accepted,https://github.com/apache/incubator-streampipes/pull/59,
 https://github.com/apache/incubator-streampipes,7ba9213012b72ea70edc6848c18ba6d90ff2c534,streampipes-connect,org.apache.streampipes.connect.adapter.format.json.xml.XmlTest.parseEventParkingFaciltyStatus,ID,Accepted,https://github.com/apache/incubator-streampipes/pull/59,
 https://github.com/apache/incubator-wayang,b7b6c1ebf65b58f5c03ac2b9fd4c1846ea877d01,wayang-platforms.wayang-graphch,org.apache.wayang.graphchi.operators.GraphChiPageRankOperatorTest.testExecution,ID,,,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.log.snapshot.MetaSimpleSnapshotTest.testSerialize,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.log.snapshot.PullSnapshotTaskTest.testWithRetry,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.ClusterPlanExecutorTest.testGetAllStorageGroupNodes,ID,,,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.AssignPathManagedMergeReaderTest.testMultManagerMergeRemoteSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testAsyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.query.reader.mult.RemoteMultSeriesReaderTest.testSyncMultSeriesReader,ID,Opened,https://github.com/apache/iotdb/pull/4380,
-https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.server.member.MetaGroupMemberTest.testRemoteAddNode,ID,,,
+https://github.com/apache/iotdb,25a98ee165131047cda93dc92203db2ab9aecbc8,cluster,org.apache.iotdb.cluster.log.snapshot.MetaSimpleSnapshotTest.testSerialize,ID,Deleted,https://github.com/apache/iotdb/pull/4380,https://github.com/apache/iotdb/commit/11c6c331e61a127a3c66906d5dfde75069ec04c6
```

Comment:
Please don't change the status for this prior PR, unless you understand whether it was `Rejected` or `Accepted` before the test was deleted. https://github.com/apache/iotdb/pull/4380 itself shows `Closed`, but at the bottom of the discussion is some `Merged`.

Reply:
I have reverted my changes and updated the Deleted tests with no prior PRs.